.github/pull_request_template.md

```diff
@@ -0,0 +1,30 @@
+<!-- Provide a general summary of your changes in the Title above -->
+
+### Description
+<!--
+Describe your changes in detail.
+For contribution process guide, look into CONTRIBUTING.md
+
+If this is a relatively large or complex change:
+ - BEFORE creating a PR, find existing or start a new discussion on Issues tab,
+ - if the discussion is concluded, go ahead with this PR,
+ - perhaps describe what alternatives you considered.
```

Comment:
please add "if the PR contains multiple commits, please squash the all that are dependent on each other. The good practice could be e.g. to keep code changes in one commit and changes in documentation in other." and "single PR should implement a single fix or enhancement - if this is not true please split PR into multiple, smaller ones"

Reply:
heh, some of these are probably for a different discussions, but I updated the template and the Contrib, a little